src/lightning_app/runners/cloud.py

```diff
@@ -157,14 +160,21 @@ def dispatch(
                 enable_app_server=app_spec.enable_app_server,
                 flow_servers=app_spec.flow_servers,
                 image_spec=app_spec.image_spec,
+                cluster_id=cluster_id,
                 works=[V1Work(name=work_req.name, spec=work_req.spec) for work_req in work_reqs],
                 local_source=True,
                 dependency_cache_key=app_spec.dependency_cache_key,
             )
+            if cluster_id is not None:
+                self._ensure_cluster_project_binding(project.project_id, cluster_id)
+
             lightning_app_release = self.backend.client.lightningapp_v2_service_create_lightningapp_release(
                 project.project_id, lightning_app.id, release_body
             )
 
+            if cluster_id is not None:
+                print(f"running app on {lightning_app_release.cluster_id}")
```

Comment:
Let's use logger and provide a better message to the users.

Reply:
good point, changed to logger.